book-worm-2010/book-worm-fe
# Feature/bookmark style

### What does this PR do? 
>Clearly and concisely describe the feature. 
This PR adds styling to the detailed book page. It also styles the forms when the buttons are clicked.

### Is this a feature or Fix/Refactor?  
Feature

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?
`src/BookDetails/_BookDetails.scss`
`src/BookDetails/BookDetails.js`
`src/Card/_Card.scss`
`src/FinishBookForm/FinishBookForm.js`
`src/FinishBookForm/_FinishBookForm.scss`
`src/NewBookMarkForm/NewBookMarkForm.js`
`src/NewBookMarkForm/_NewBookMarkForm.scss`
`src/StarRating/StarRating.js`
`src/StarRating/StarRating.scss`
`src/style/_variables.scss`

### How should this be tested?
>What steps need to be taken to confirm functionality? 
1. `git fetch` branch
2. run `npm start`
3. Click on the buttons to see the forms

### Steps To Take
>Any questions and possible steps moving forward.
Need to connect buttons to a working backend endpoint.
